app/agents/voice/automatic/services/filters/krisp/noise.py

```
@@ -0,0 +1,124 @@
+
+    async def filter(self, audio: bytes) -> bytes:
+        # Return unfiltered audio if filtering is disabled or krisp is not functional
+        if not self._filtering or not self._krisp_functional or not self._session:
+            return audio
+
+        # Krisp filtering logic
+        samples = np.frombuffer(audio, dtype=np.int16)
+        if samples.size % self._samples_per_frame:
+            raise ValueError(
+                f"Audio length {samples.size} samples is not a multiple of {self._samples_per_frame}"
+            )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)